### DISCUSSION TITLE
v0.3.0 ?

### DISCUSSION THREAD

**Discussion by cailloumajor** [vuejs/pinia#434] [Help and Questions] [answered]
Hello, version 0.3.0 has been released https://www.npmjs.com/package/pinia/v/0.3.0, but I find no trace of it in the repository on GitHub, despite having looked in all branches.

**Comment by posva** [comment, marked as answer]
I had a yarn error while publishing and my release process didn't finish so I didn't push the tag and the last commits on the v1 branch 🤡 

I will push them on Monday alondside a new release with #429  fix!

**Reply by posva** [reply]
In the end I just went to the office to push the missing changes because I want to push other changes this weekend
